src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToHtml5LocalDateTimeTransformer.php

```diff
@@ -23,6 +23,19 @@
 class DateTimeToHtml5LocalDateTimeTransformer extends BaseDateTimeTransformer
 {
     public const HTML5_FORMAT = 'Y-m-d\\TH:i:s';
+    public const HTML5_WITHOUT_SECONDS_FORMAT = 'Y-m-d\\TH:i';
+
+    private readonly bool $withSeconds;
+
+    /**
+     * @param bool $withSeconds
+     */
+    public function __construct(string $inputTimezone = null, string $outputTimezone = null/* , private readonly bool $withSeconds = true */)
```

Comment:
since this is the constructor adding a real argument is okay

Reply:
Does it still need to be added to the CHANGELOG? :thinking:

Reply:
@xabbuh is right, and yes, it's useful to have it in the CHANGELOG.